MapboxCoreNavigation/Route.swift

```diff
@@ -2,7 +2,7 @@ import MapboxDirections
 
 extension Route {
     
-    public var json: String? {
+    public func jsonRepresentation() -> String? {
```

Comment:
@1ec5 - Is this what you had in mind? Does the naming of the function seem appropriate?

Reply:
Something like that, thanks. But I still think the method should be private since it’s such a specific implementation detail. Any application could trivially implement the same functionality if needed, and not just for Route but for RouteLeg, Intersection, etc.